tests/checks/fixtures/kubernetes/events.json

```diff
@@ -0,0 +1,532 @@
+        "namespace": "default",
+        "selfLink": "/api/v1/namespaces/default/events/hello-node-47289321.14527a0ef18f9373",
+        "uid": "44019fcc-2429-11e6-b9c2-42010af00026",
+        "resourceVersion": "3988",
+        "creationTimestamp": "2016-05-27T16:37:13Z",
+        "deletionTimestamp": "2016-05-27T17:37:13Z"
+      },
+      "involvedObject": {
+        "kind": "ReplicaSet",
+        "namespace": "default",
+        "name": "hello-node-47289321",
+        "uid": "e6161e98-1c0f-11e6-b9c2-42010af00026",
+        "apiVersion": "extensions",
+        "resourceVersion": "265432"
+      },
+      "reason": "SuccessfulCreate",
+      "message": "Created pod: hello-node-47289321-lr676",
+      "source": {
+        "component": "replicaset-controller"
+      },
+      "firstTimestamp": "2016-05-27T16:37:13Z",
+      "lastTimestamp": "2016-05-27T16:37:13Z",
+      "count": 1,
+      "type": "Normal"
+    },
+    {
+      "metadata": {
+        "name": "hello-node-47289321.14527a0ef2424590",
+        "namespace": "default",
+        "selfLink": "/api/v1/namespaces/default/events/hello-node-47289321.14527a0ef2424590",
+        "uid": "44043ba8-2429-11e6-b9c2-42010af00026",
+        "resourceVersion": "3989",
+        "creationTimestamp": "2016-05-27T16:37:13Z",
+        "deletionTimestamp": "2016-05-27T17:37:13Z"
+      },
+      "involvedObject": {
+        "kind": "ReplicaSet",
+        "namespace": "default",
+        "name": "hello-node-47289321",
+        "uid": "e6161e98-1c0f-11e6-b9c2-42010af00026",
+        "apiVersion": "extensions",
+        "resourceVersion": "265432"
+      },
+      "reason": "SuccessfulCreate",
+      "message": "Created pod: hello-node-47289321-91tfd",
+      "source": {
+        "component": "replicaset-controller"
+      },
+      "firstTimestamp": "2016-05-27T16:37:13Z",
+      "lastTimestamp": "2016-05-27T16:37:13Z",
+      "count": 1,
+      "type": "Normal"
+    },
+    {
+      "metadata": {
+        "name": "hello-node-47289321.14527a0ef2e10858",
+        "namespace": "default",
+        "selfLink": "/api/v1/namespaces/default/events/hello-node-47289321.14527a0ef2e10858",
+        "uid": "4405edc4-2429-11e6-b9c2-42010af00026",
+        "resourceVersion": "3990",
+        "creationTimestamp": "2016-05-27T16:37:13Z",
+        "deletionTimestamp": "2016-05-27T17:37:13Z"
+      },
+      "involvedObject": {
+        "kind": "ReplicaSet",
+        "namespace": "default",
+        "name": "hello-node-47289321",
+        "uid": "e6161e98-1c0f-11e6-b9c2-42010af00026",
+        "apiVersion": "extensions",
+        "resourceVersion": "265432"
+      },
+      "reason": "SuccessfulCreate",
+      "message": "Created pod: hello-node-47289321-a3x8n",
+      "source": {
+        "component": "replicaset-controller"
+      },
+      "firstTimestamp": "2016-05-27T16:37:13Z",
+      "lastTimestamp": "2016-05-27T16:37:13Z",
+      "count": 1,
+      "type": "Normal"
+    }
+  ]
+}
```

Comment:
This fixture is a little bit large but contains a different kind of events, could be useful later.